marketplace/deployer_util/set_ownership.py

```diff
@@ -140,6 +140,7 @@ def dump(outfile, resources, included_kinds, app_name, app_uid,
       # https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents
       log.info("Application '{:s}' does not own cluster-scoped '{:s}/{:s}'",
                app_name, resource["kind"], resource["metadata"]["name"])
+      to_be_dumped.append(resource)
```

Comment:
optional: to make this clearer (and avoid something like this), it's probably better to extract the content of the `for` loop into a transforming function, which could live within the scope of this `dump()` function, and rewrite this as:

```python
to_be_dumped = [maybe_assign_ownership(resource) for resource in resources]
```

Reply:
Done.